ios/RCTWebRTC/WebRTCModule+VideoTrackAdapter.h

```diff
@@ -0,0 +1,13 @@
+
+#import "WebRTCModule.h"
+#import <WebRTC/RTCPeerConnection.h>
+
+@interface RTCPeerConnection (VideoTrackAdapter)
+
+@property (nonatomic, strong) NSMutableDictionary<NSString *,  id> *videoTrackAdapters;
+
+- (void)addVideoTrackAdapter:(NSString*)streamReadtId track:(RTCVideoTrack*)track;
```

Comment:
what does "streamReadtId" stand for ? is there a typo ?

Reply:
It is supposed to be `streamReactId`, it's a typo. Will fix.